packages/experience-builder-sdk/src/hooks/useDetectEditorMode.tsx

```diff
@@ -35,8 +39,20 @@ export const useDetectEditorMode = ({ isClientSide = false }: UseDetectEditorMod
             window.__EB__ = {};
           }
           window.__EB__.isEditorMode = true;
-          window.removeEventListener('message', onMessage);
+          // window.removeEventListener('message', onMessage);
+        }
+      }
+
```

Comment:
Introducing a new event sent from the web app, that will set a flag similar to the editor mode to set it in read-only mode. 
We will also set the mode to editor mode, so that we enable the iframe communication and don't fetch staff over the network